### DISCUSSION TITLE
Support image messages in the playground

### DISCUSSION THREAD

**Discussion by jeff-an** [langfuse/langfuse#4268] [Ideas]
### Describe the feature or potential improvement

Many other LLM observability platforms have the ability to add messages that contain images in the playground.

The UX I've seen usually allows the user to either paste in an image URL, or drop a file from their own machine onto the message. Presumably the platform then converts the file to base 64 and puts the value into the message.

Almost all of our agents require images, so this would be a great way to rapidly iterate on prompts. Today, we would have to run a script to extract the base64 from an image we have on our machine and then paste that giant block into the frontend. The frontend today also doesn't seem to support messages with type image at all.

**Comment by marcklingen** [comment]
thanks for the feedback!

**Reply by lukepighetti** [reply]
this still not a thing? thought I was missing something when I was trying it out today

**Comment by yingzhuivy** [comment]
any update on this feature? really need it

**Comment by metalik** [comment]
+1

**Comment by jaco4873** [comment]
+1 :)

**Comment by eggachecat** [comment]
+1

**Comment by skcadri** [comment]
+1

**Comment by salqadri** [comment]
Any ETA on this?

**Comment by aoi1** [comment]
+1

**Comment by shujiogiri** [comment]
+1

**Comment by michael-j-chinn** [comment]
Is this a duplicate of https://github.com/orgs/langfuse/discussions/6017? Should we combine them so we can better represent the demand for this feature?

**Comment by FaderYin** [comment]
+1

**Comment by stg609** [comment]
+1

**Comment by Baicheng-MiQ** [comment]
+1

**Comment by royp-duzz** [comment]
+1000

**Comment by filipegl** [comment]
+10000000
